src/bindings/go-elektra/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+find_program (GO_EXECUTABLE go)
+
+if (GO_EXECUTABLE)
+	add_binding (go-elektra)
+
+	# create temporary elektra.pc file
+	configure_file ("${CMAKE_CURRENT_SOURCE_DIR}/temp.elektra.pc" "${CMAKE_CURRENT_BINARY_DIR}/elektra.pc" @ONLY)
+
+	# build
+	add_custom_command (
+		OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/lib/libelektra.golib"
+		COMMAND ${CMAKE_COMMAND} ARGS -E env GO111MODULE=on ${GO_EXECUTABLE} build ./kdb
```

```suggestion
		COMMAND ${CMAKE_COMMAND} ARGS -E env PKG_CONFIG_PATH=${CMAKE_CURRENT_BINARY_DIR} GO111MODULE=on ${GO_EXECUTABLE} build ./kdb
```

To use the `elektra.pc` you generate above, you now need to add the `PKG_CONFIG_PATH` here.